src/Models/Models.ts

```diff
@@ -12,8 +12,9 @@ import Transitions from './Transition';
 import Instance from './Instance';
 import Stage from './Stage';
 import TaxRate from './TaxRate';
+import Event from './Event';
 
-type Models = User | Business | Mailbox | Product | Template | Transitions | Stage | Instance | Permission | Partner | Partnership | DocType;
+type Models = User | Business | Mailbox | Product | Template | Transitions | Stage | Instance | Permission | Partner | Partnership | DocType | Event;
 
```

Comment:
This is going to get messy.

Reply:
Thing Callum fixed this in `feat/billing` but thats still an active branch... Think I might steal it and rebase master into the active branches